CITATION.cff

```diff
@@ -1,45 +1,71 @@
-# YAML 1.2
----
-abstract: "The new software FEniCS-preCICE is a middle software layer, sitting in between the existing finite-element library FEniCS and the coupling library preCICE. The middle layer simplifies coupling (existing) FEniCS application codes to other simulation software via preCICE. To this end, FEniCS-preCICE converts between FEniCS and preCICE mesh and data structures, provides easy-to-use coupling conditions, and manages data checkpointing for implicit coupling. The new software is a library itself and follows a FEniCS-native style. Only a few lines of additional code are necessary to prepare a FEniCS application code for coupling. We illustrate the functionality of FEniCS-preCICE by two examples: a FEniCS heat conduction code coupled to OpenFOAM and a FEniCS linear elasticity code coupled to SU2. The results of both scenarios are compared with other simulation software showing good agreement."
-authors: 
-  -
-    affiliation: "Technical University of Munich"
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: FEniCS-preCICE
+message: >-
+  If you use this software, please cite it using the
+  metadata from this file.
```

Comment:
We can also use this message for providing some further guidance. E.g. something similar to https://github.com/precice/.github/blob/9c7649853e4ca65c9961e655e8a808848efe6163/profile/README.md?plain=1#L15

Reply:
I added the following: "When using or referring to preCICE in academic publications, please follow the [citation guidelines](https://precice.org/fundamentals-literature-guide.html)."